caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+
+.slider-card {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  gap: 16px;
+  transform: translateX(0);
+  transition: transform 0.3s ease-out;
+}
+
+/* To prevent scrolling on the cards */
+.slider-card-wrapper {
+  overflow: hidden;
+}
+
+.card {
+  flex: 1 0 100%;
+  height: auto;
+  width: 100%;
+}
+
+.card .short-description {
+  padding: 16px;
+  border: 1px solid var(--highlight-background-color);
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgb(0 0 0 / 10%);
+  height: 180px;
+}
+
+.card .short-description.hide {
+  display: none;
+}
+
+.close-button {
+  background: url("../../icons/close.svg") no-repeat;
+  min-width: 20px;
+  min-height: 20px;
+  transform: scale(0.8);
+  float: right;
+  margin-bottom: 10px;
+  cursor: pointer;
+}
+
+.close-button.hide {
+  display: none;
+}
+
+.card .short-description .title {
+  font-family: var(--text-font-family);
+  font-weight: var(--emphasized-font-weight);
+  font-size: 18px;
+  text-transform: uppercase;
+}
+
+.show-more {
```

Comment:
All the CSS classes should be scoped.
Example of scoping to the block: https://github.com/hlxsites/caesars/blob/main/caesars-palace/blocks/carousel/carousel.css

Reply:
Done.